### PR TITLE
Remove duped PyPI section in install.rst

### PR DESCRIPTION
**Pull request by C0rn3j**
It's there twice.

https://docs.aiogram.dev/en/dev-3.x/install.html#id1

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -2,13 +2,6 @@
 Installation
 ############
 
-From PyPI
----------
-
-.. code-block:: bash
-
-    pip install -U aiogram
-
```
Comment (Olegt0rr) [inline docs/install.rst:11]: Why you delete first? Delete second :) It's preferred way

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

File: docs/install.rst (hunk 1 above)
Comment (C0rn3j) [reply docs/install.rst:11]: Had to add a section then

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

File: docs/install.rst [hunk 2]
```diff
@@ -32,3 +18,14 @@ From GitHub
 .. code-block:: bash
 
     pip install https://github.com/aiogram/aiogram/archive/refs/heads/dev-3.x.zip
+
+Stable build (2.x)
```
[the remark this answers is not in the document]
Comment (C0rn3j) [reply docs/install.rst:22]: Does that mean `Development build (3.x)` no longer applies as that is the current stable?

[the remark this answers is not in the document]
Comment (Olegt0rr) [reply docs/install.rst:22]: stable - from pypi
developement - from github

**Comment by JrooTJunior** [comment]
Fixed by https://github.com/aiogram/aiogram/commit/46017f814d4e532390597c29ca9273a3da9c1f88

**Comment by C0rn3j** [comment]
Thanks!

Arch Linux has updated the package now, so the warning is not necessary anymore.

https://gitlab.archlinux.org/archlinux/packaging/packages/python-aiogram/-/commit/4417295e318b767d0d12b8b4e837b9dbe66fabe1

**Pull request closed without merging**